test/Integration/TestDocs/README.md

```diff
@@ -0,0 +1,6 @@
+# Steps to Compile & Execution of SDV automation suit 
```

Comment:
SDV automation suit  -> integration tests.  
@m1093782566 , Should we move any docs to the kubeedge/docs folder and create a sub-folder for integration-tests ?